code/go/0chain.net/smartcontract/dbs/event/challenge.go

```diff
@@ -141,9 +141,6 @@ func (edb *EventDb) GetOpenChallengesForBlobber(blobberID string, from int64, li
 		Order(clause.OrderByColumn{
 			Column: clause.Column{Name: "round_created_at"},
 			Desc:   true,
-		}).
```

Comment:
you cannot remove ordering by id, pagination will be inconsistent then

Reply:
Actually round_created_at is already unique so it is redundant now. We had this earlier when we were using created_at  which had multiple entries on same created_at  because it was in second and there are multiple challenges per second.
So I think it is safe to remove.